indico/modules/rb_new/client/js/components/BookingBootstrapForm.jsx

```diff
@@ -20,9 +20,8 @@ import moment from 'moment';
 import PropTypes from 'prop-types';
 import React from 'react';
 import {Button, Form, Select} from 'semantic-ui-react';
-import RCCalendar from 'rc-calendar';
-import RangeCalendar from 'rc-calendar/lib/RangeCalendar';
-import DatePicker from 'rc-calendar/lib/Picker';
+import SingleDatePicker from 'indico/react/components/SingleDatePicker';
```

Comment:
I'd export those in `indico/react/components/index.js` so you can import them directly from `indico/react/components`

Reply:
Any idea why doing this you mention causes my `dates.scss` not being loaded?